packages/components/src/font-size-picker/index.tsx

```diff
@@ -46,6 +46,7 @@ const UnforwardedFontSizePicker = (
 		size = 'default',
 		units: unitsProp = DEFAULT_UNITS,
 		value,
+		valueMode = 'literal',
```

Comment:
Is there a condition for which this should be `'slug'`?

Reply:
It's `literal` when you use a custom font size that is not a preset, but it's `slug` when you use a preset. Previously the font size was only working with `size`, so you were not able to select a different font size if the `size` property held the same value, even if it was a fixed vs. a fluid value. Now it differentiates on the slugs of you use a preset, so when using for example TT5 it can hold `small`, `medium`, `large` and so on, and custom sizes well you can add via the typography interface.